.github/travis/script.sh

```diff
@@ -22,14 +22,18 @@ export TRAVIS_TAG=${TRAVIS_TAG:-$(git describe --match=v*)}
 BAZEL_CXXOPTS=(--cxxopt=-std=c++17)
 
 case "$MODE" in
-compile-n-test)
-    bazel test --noshow_progress "${BAZEL_CXXOPTS[@]}" //...
-    bazel build -c opt --noshow_progress "${BAZEL_CXXOPTS[@]}" //...
+test)
+    bazel test --noshow_progress --curses=no --show_timestamps "${BAZEL_CXXOPTS[@]}" //...
```

Comment:
Just noticed: `build` and `test` should use the same options, both should use `-c opt`.  Different options will double the work due to bazel cache missing.  If the old `compile-n-test` added the missing `-c opt`, the second command should be much faster, I wager nearly instantaneous because of cache hits.  Can we try this experiment first?  

`test` should be a superset of `build`.  One could argue for removing `build` altogether and letting `test` do all of the work.

Reply:
@fangism note that for `-c opt` means build with optimization enabled and with `assert()` calls disabled (`-O2 -DNDEBUG`)

So the decision here is whether we want to disable asserts for testing. I think it would be beneficial to leave this as is at least for PRs (although it would be better if it stayed on for master builds too) - the `-c opt` flag is really only needed for creating releases to make the binaries smaller.

Reply:
It doesn't help: https://travis-ci.com/github/antmicro/verible/jobs/426145668

Reply:
So Verible's codebase doesn't use any traditional `assert`s from `<cassert>`.  It uses `CHECK*` for assertions, which are always-on.

Reply:
ack, thanks!

Reply:
At line 1147 of https://travis-ci.com/github/antmicro/verible/jobs/426145668

```
(09:30:53) INFO: Build options --cxxopt, --define, and --host_cxxopt have changed, discarding analysis cache.
```

If you see that, that may be an early warning sign of cache-missing a lot.  Can you verify that the options match?
Or could we eliminate `build` if `test` is a superset anyway?

Reply:
Could we reduce this to `test -c opt` which is a superset of `build`?  We need to fix CI soon.